index.html

```diff
@@ -239,13 +239,15 @@ <h1>Skills</h1>
                 </form>
             </section>
         </section>
+        <footer>
         <ul class="social_form">
             <li><a href="#" class="fa fa-facebook-square"></a></li>
             <li><a href="#" class="fa fa-twitter-square"></a></li>
             <li><a href="#" class="fa fa-github-square"></a></li>
             <li><a href="#" class="fa fa-instagram"></a></li>
         </ul>
         <h2 class="eaddress">anas599@yahoo.com</h2>
+        </footer>
```

Comment:
- [ ] nice work with the desktop version, kindly add the background image to the contact section of the desktop version so it matches the Figma design
- [ ] add padding to the input placeholders as well so as to follow the Figma design
yours
![Screenshot (649)](https://user-images.githubusercontent.com/65298048/206552900-a360b248-cf61-41bd-97b3-08618b03b25f.png)

figma design
![Screenshot (650)](https://user-images.githubusercontent.com/65298048/206553097-b63a0d38-6b31-4cb4-987e-859ee9416035.png)